waspc/README.md

```diff
@@ -17,150 +17,135 @@ If you would like to make your first contribution, here is a handy checklist we
 
 ## Quick overview
 
-Wasp compiler is implemented in Haskell, but you will also see a lot of Javascript and other web technologies because Wasp compiles it's own code into them.
+The core of `waspc` is implemented in Haskell, but you will also see a lot of Javascript/TypeScript and other web technologies since Wasp generates web apps in those.
 
-You don't have to be expert in Haskell to contribute or understand the code, since we don't use complicated Haskell features much -> most of the code is relatively simple and straight-forward, and we are happy to help with the part that is not.
+You don't have to be expert in Haskell to contribute or understand the code, since we don't use complicated Haskell features much -> most of the code is relatively simple and straight-forward, and we are happy to help with any parts that are not.
 
-Main result of building the project is `wasp` executable (also referred to as CLI), which is both Wasp compiler, CLI and Wasp project runner in one - one tool for everything Wasp-related.
+Main result of building the project is `wasp` executable (also referred to as CLI), which is both Wasp compiler, CLI and Wasp project runner in one - one main tool for everything Wasp-related.
 
-`wasp` executable takes `.wasp` files and `ext/` dir as input and generates a web app from them.
-
-![Wasp compilation](images/wasp-diagram.png)
-
-It can then also run that web app for you, deploy it (not yet but that is coming), and manage it in other ways.
```

Comment:
I removed this because this section is now about what Wasp itself does, but about what technical details of how `waspc` is implemented.
There is actually another section more below that again repeats this information, so one more reason to remove it.